ui/app/AppLayouts/Chat/panels/UserListPanel.qml

```diff
@@ -54,17 +55,48 @@ Item {
         }
         boundsBehavior: Flickable.StopAtBounds
         model: usersModule.model
-        delegate: UserDelegate {
-            publicKey: model.id
-            name: model.name
-            icon: model.icon
-            isAdded: model.isAdded
-            userStatus: model.onlineStatus
-            messageContextMenu: root.messageContextMenu
-            isCurrentUser: rootStore.isCurrentUser(model.id)
-        }
         section.property: "onlineStatus"
         section.delegate: (root.width > 58) ? sectionDelegateComponent : null
+        delegate: StatusMemberListItem {
+            anchors.left: parent.left
+            anchors.leftMargin: 8
+            anchors.right: parent.right
+            anchors.rightMargin: 8
+            nickName: model.localNickname
+            userName: model.name
+            chatKey: model.id
+            trustIndicator: model.trustIndicator
+            isMutualContact: model.isMutualContact
+            image.source: {
+                if ((!model.isAdded &&
+                    Global.privacyModuleInst.profilePicturesVisibility !==
+                    Constants.profilePicturesVisibility.everyone)) {
+                    return "";
+                }
+                return model.icon;
+            }
+            image.isIdenticon: model.isIdenticon
+
+            isOnline: model.onlineStatus
+            icon.color: Theme.palette.userCustomizationColors[Utils.colorIdForPubkey(model.id)]
+            ringSettings.ringSpecModel: Utils.getColorHashAsJson(model.id)
+            onClicked: {
+                if (mouse.button === Qt.RightButton) {
+                    // Set parent, X & Y positions for the messageContextMenu
+                    messageContextMenu.parent = this
+                    messageContextMenu.setXPosition = function() { return 0}
+                    messageContextMenu.setYPosition = function() { return height }
```

Comment:
So far we've positioned context menus always 4px below the cursor. We probably want to do the same here to stay consistent

Reply:
@PascalPrecht sure thing, updated!